### PR TITLE
First attempt at the forking banner

### PR DESCRIPTION
**Pull request by bconfortin**
https://github.com/AugurProject/augur/issues/3933

Opened a PR just to see if this is making sense

### CODE REVIEW

File: packages/augur-ui/src/modules/reporting/forking.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        </span>
+        <div>
+          {props.hasStakedRep && <SecondaryButton action={() => {}} text={'Release my REP'} />}
+          {props.hasRepBalance && <PrimaryButton action={() => {}} text={'Migrate REP'} />}
+        </div>
+      </div>
+      <div>
+        <CountdownProgress
+          label={'Time remaining in Fork Window'}
+          time={formatTime(1577232000)}
```
Comment (bthaile) [inline packages/augur-ui/src/modules/reporting/forking.tsx]: pass values in from container. put fake numbers in container where they will be set later.

File: packages/augur-ui/src/modules/reporting/forking.tsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+import Styles from "modules/reporting/forking.styles.less";
+import {ExclamationCircle} from "modules/common/icons";
+import {CountdownProgress, formatTime} from "modules/common/progress";
+import {PrimaryButton, SecondaryButton} from "modules/common/buttons";
+
+interface ForkingProps {
+  header?: string;
+  subheader?: string;
+  hasStakedRep?: boolean;
+  hasRepBalance?: boolean;
```
Comment (bthaile) [inline packages/augur-ui/src/modules/reporting/forking.tsx]: add the time properties to the props

both `time` and `currentTime` will be `DateFormattedObject`

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://featurepeek.com). 🎊

 This link will always point to this PR's latest changes: https://peek.run/jvgh4z6

**Comment by bthaile** [comment]
made some changes, wired up the buttons. wired it up to universe.forkingInfo

image

**Review by bthaile — APPROVED** [review]
Lgtm, @bconfortin see what you think. Id like to get this merged up and we can continue to test

**Comment by bconfortin** [comment]
> Lgtm, @bconfortin see what you think. Id like to get this merged up and we can continue to test

@bthaile I just went through all of your commits and the code seems solid. Haven't tested it yet. Awesome job :clap:

**Pull request merged**